### DISCUSSION TITLE
每日签door

### DISCUSSION THREAD

**Discussion by ldqk** [52ABP/LTMCompanyNameFree.YoyoCmsTemplate#58] [General 聊天]
> 请规范发图。。。已删。

**Comment by ltm0203** [comment]
这个就
![16108E65](https://user-images.githubusercontent.com/7875527/138415359-d0391ee0-aab5-4126-9fa1-2cbf241bfe76.png)
牛批了。。。

**Comment by jackzhum** [comment]
我眼瞎了

**Comment by zhuhuanzi** [comment]
666
